gradle/kotlin-mpp-compose-lib.gradle

```diff
@@ -68,7 +68,7 @@ kotlin {
         }
     }
 
-    android() {
+    androidTarget() {
```

Comment:
In Kotlin 1.9.20, the target name for Android is changed from `android` to `androidTarget`.